tests/attacks/evasion/test_dpatch.py

```diff
@@ -16,16 +16,16 @@
 # TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 import logging
+
 import numpy as np
 import pytest
+from tests.attacks.utils import backend_test_classifier_type_check_fail
+from tests.utils import master_seed
```

Comment:
I think we should keep the imports from `tests` after the imports from `art` because they are not third-party and to keep it consistent across all tests.